bounties_api/notifications/notification_client.py

```diff
@@ -127,16 +141,20 @@ def fulfillment_updated(
             str(notifications['FulfillmentUpdatedIssuer']),
             notification_name=notifications['FulfillmentUpdatedIssuer'],
             user=bounty.user,
-            data=string_data_issuer,
-            subjet='Submission was Updated',
+            from_user=fulfillment.user,
+            string_data=string_data_issuer,
+            bounty_title=bounty.title,
+            subject='Submission was Updated',
```

Comment:
This wasn't part of your change, but it seems like weird capital casing. Is this intentional @villanuevawill ?

Reply:
@codeluggage i think the reasoning behind it is that the subject field is used to populate the subject field in an email that is sent out to the user--therefore, it is capitalized like a title